src/main/java/org/sonarsource/sonarlint/ls/SonarLintExtendedLanguageClient.java

```diff
@@ -73,13 +75,13 @@ class ShowRuleDescriptionParams {
     @Expose
     private final String htmlDescription;
     @Expose
-    private final String type;
+    private final RuleType type;
     @Expose
-    private final String severity;
+    private final IssueSeverity severity;
```

Comment:
Here we expose fields to be shared with the client, will this work for those enums ? Probably yes, it might be worth testing